### PR TITLE
Fixing Linting issues

### PR DESCRIPTION
**Pull request by AbdulRJacob**
Removed trailing whitespaces and unused imports

Fixes: #32

### CODE REVIEW

File: nlp.py [hunk 1]
```diff
@@ -18,12 +17,11 @@ def extract_characters(text):
 
     try:
         nlp = spacy.load("en_core_web_md")
-    except:
+    except Exception:
```
Comment (zeegtab) [inline nlp.py:20]: Can we be more specific about what kind of Exception we should be catching, e.g RuntimeError, or will the linter complain?

**Review by zeegtab — COMMENTED** [review]
Looks cool! I just had a question about Exceptions

File: nlp.py (hunk 1 above)
Comment (AbdulRJacob) [reply nlp.py]: Done :)

**Review by BOluokun — APPROVED** [review]
LGTM!

**Pull request merged**